code/modules/mob/living/silicon/ai/ai.dm

```diff
@@ -673,7 +673,7 @@ var/list/ai_list = list()
 
 	if(stat == 2)
 		return //won't work if dead
-	var/list/ai_emotions = list("Very Happy", "Happy", "Neutral", "Unsure", "Confused", "Sad", "BSOD", "Blank", "Problems?", "Awesome", "Facepalm", "Friend Computer", "Dorfy", "Blue Glow", "Red Glow","Flowey","Flowey Grin")
+	var/list/ai_emotions = list("Very Happy", "Happy", "Neutral", "Unsure", "Confused", "Sad", "BSOD", "Blank", "Trollface", "Awesome", "Facepalm", "Friend Computer", "Dorfy", "Blue Glow", "Red Glow", "Flowey", "Flowey Grin")
```

Comment:
Ah I see you've replaced "Problems?" with "TrollFace", but I suspect there'll still be issues with "Friend Computer", etc.